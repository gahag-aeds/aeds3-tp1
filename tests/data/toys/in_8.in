9
0.0000 0.8767 0.12728 0.86358 0.68728 0.01322 0.05774 0.28517 0.67794 
0.1233 0.0000 0.95523 0.17773 0.38851 0.29496 0.63434 0.06337 0.04166 
0.87272 0.04477 0.0000 0.60679 0.71852 0.48513 0.90542 0.51005 0.07873 
0.13642 0.82227 0.39321 0.0000 0.98536 0.11967 0.0541 0.80692 0.40943 
0.31272 0.61149 0.28148 0.01464 0.0000 0.82698 0.97125 0.32758 0.7256 
0.98678 0.70504 0.51487 0.88033 0.17302 0.0000 0.65923 0.79653 0.10246 
0.94226 0.36566 0.09458 0.9459 0.02875 0.34077 0.0000 0.27064 0.25501 
0.71483 0.93663 0.48995 0.19308 0.67242 0.20347 0.72936 0.0000 0.46054 
0.32206 0.95834 0.92127 0.59057 0.2744 0.89754 0.74499 0.53946 0.0000 
