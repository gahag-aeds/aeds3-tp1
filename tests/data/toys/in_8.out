0.076385 0.031955 0.113411 0.088871 0.122425 0.150434 0.051885 0.121187 0.243447 